editor/src/components/canvas/canvas-strategies/strategies/set-border-radius-strategy.tsx

```diff
@@ -123,6 +125,16 @@ export const setBorderRadiusStrategy: CanvasStrategyFactory = (
       ? 'individual'
       : 'all'
 
+  const addOverflowHiddenCommand = addPropIfDoesntExist(
+    'always',
+    selectedElement,
+    PP.create('style', 'overflow'),
+    'hidden',
+    'Element now hides overflowing content',
+  )
```

Comment:
Have you checked how hard it would be to do the same thing by composing `setProperty` and `showToastCommand`, instead of creating a bespoke command?